src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -645,6 +645,20 @@ public enum ClassLiteralsMode {
     ALL
   }
 
+  /**
+   * Whether to use literals specified via the {@code --literals-file} command-line option or
```

Comment:
I don't understand the "or".  Is it a logical or, or is it what happens if the value is false?  Please clarify.